drivers/flash/Kconfig

```diff
@@ -18,7 +18,7 @@ menuconfig FLASH
 
 config FLASH_PAGE_LAYOUT
 	bool "API for retrieving the layout of pages"
-	depends on FLASH
+	depends on FLASH && SOC_FLASH_NRF5 && SOC_FLASH_STM32
```

Comment:
Shouldn't this be:

depends on FLASH && (SOC_FLASH_NRF5 || SOC_FLASH_STM32)

?

Reply:
yes - I was have done this listening to your presentation :)